Common/src/main/java/com/tabnineCommon/chat/commandHandlers/context/workspace/FindSymbolsCommandExecutor.kt

```diff
@@ -2,11 +2,28 @@ package com.tabnineCommon.chat.commandHandlers.context.workspace
 
 import com.intellij.openapi.editor.Editor
 import com.intellij.openapi.project.Project
+import com.tabnineCommon.chat.commandHandlers.utils.StringCaseConverter
 import com.tabnineCommon.chat.commandHandlers.utils.SymbolsResolver
+import java.util.stream.Stream
+import kotlin.streams.toList
+
+private const val MAX_RESULTS_PER_SYMBOL = 5
 
 class FindSymbolsCommandExecutor : CommandsExecutor {
     override fun execute(arg: String, editor: Editor, project: Project): List<String> {
-        return SymbolsResolver.resolveSymbols(project, editor.document, arg, 5)
+        val camelCaseArg = StringCaseConverter.toCamelCase(arg)
+        val snakeCaseArg = StringCaseConverter.toSnakeCase(arg)
+        val camelCaseSymbols = SymbolsResolver.resolveSymbols(
+            project, editor.document, camelCaseArg,
+            MAX_RESULTS_PER_SYMBOL
+        )
+        val snakeCaseSymbols = SymbolsResolver.resolveSymbols(
+            project, editor.document, snakeCaseArg,
+            MAX_RESULTS_PER_SYMBOL
+        )
+
+        return Stream.concat(camelCaseSymbols.stream(), snakeCaseSymbols.stream())
```

Comment:
i think you should  dedup that list

Reply:
i added a dedup at the end in `WorkspaceContext.kt`: `WorkspaceContext(symbols.distinct())`
its better to do it there, because duplicate symbols can arrive from different commands, for example if one command is "foo" and the other is "fooBar" - you can find "fooBar" in both.